Lib/test/make_ssl_certs.py

```diff
@@ -206,8 +206,9 @@ def print_cert(path):
     with open('ssl_key.pem', 'w') as f:
         f.write(key)
     print("password protecting ssl_key.pem in ssl_key.passwd.pem")
-    check_call(['openssl','rsa','-in','ssl_key.pem','-out','ssl_key.passwd.pem','-des3','-passout','pass:somepass'])
-    check_call(['openssl','rsa','-in','ssl_key.pem','-out','keycert.passwd.pem','-des3','-passout','pass:somepass'])
+    # pkey uses modern encryption format
```

Comment:
This probably won't be true in 5 years. Perhaps leave this comment in the commit message? It makes more sense as only a justification for this change.

Reply:
SGTM, the comment is gone.